### DISCUSSION TITLE
Oracle failing due to column name quoting

### DISCUSSION THREAD

**Discussion by valhuber** [sqlalchemy/sqlalchemy#10814] [Usage Questions]
My Oracle table has lower case column names.  The query fails as shown below, and works if I manually quote the col name as shown in diagram.  So, I am guessing I need to make SQLAlchemy quote the col names...?

I saw (a very old) post suggestion `column.quote = True`, but that seemed to have no effect.

Not clear how to address this...

<img width="1221" alt="unquoted_col_names" src="https://github.com/sqlalchemy/sqlalchemy/assets/5579631/92608512-c5fe-4279-bc1b-cbc0965f4782">

**Comment by CaselIT** [comment]
Hi,

You should set `quote=True` in the constructor of `Column`. Does it work if set like this?

**Comment by valhuber** [comment]
Yes!

I in fact suspected that, but VSCode code completion did not confirm such a parameter, nor did I spot it in the docs.  

But, the hover help was enormously helpful -- now I have a clue when to generate this into the models.

Thanks so much for your rapid reply.  You guys are awesome.
